frontend/online-shop/src/components/ProductDetail.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from "@material-ui/core";
+import { productDetailStyle } from "../styles/ProductDetailStyle";
+
+export const ProductDetail = (props: any) => {
+  const style = productDetailStyle();
+
+  const { product } = props.location.state;
+
+  return (
+    <>
+      <div>
+        <h2 className={style.headingStyle}>Product: {product.name}</h2>
+        <Button className={style.buttonStyle}> Edit </Button>
+        <Button className={style.buttonStyle}>Add </Button>
+      </div>
+
+      <div>
+        <table>
+          <tr>
+            <td>Name</td>
+            <td> {product.name}</td>
+          </tr>
+
```

Comment:
No need for spaces between elements, it just adds more lenght to the file

Reply:
Fixed.